babel.dependencies.config.js

```diff
@@ -1,31 +0,0 @@
-/** @format */
-const config = {
-	// see https://github.com/webpack/webpack/issues/4039#issuecomment-419284940
-	sourceType: 'unambiguous',
-	presets: [
-		[
-			'@babel/env',
-			{
-				modules: false,
-				useBuiltIns: 'entry',
-				corejs: 2,
-				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
-				exclude: [ 'transform-typeof-symbol' ],
-			},
-		],
-	],
-	plugins: [
-		'@babel/plugin-syntax-dynamic-import',
-		[
-			'@babel/transform-runtime',
-			{
-				corejs: false, // we polyfill so we don't need core-js
-				helpers: true,
-				regenerator: false,
-				useESModules: false,
-			},
-		],
```

Comment:
This plugin is configured the same way in `calypso-build` :heavy_check_mark: https://github.com/Automattic/wp-calypso/blob/948f23d6a0c7d9243377cb5346997238a160c183/packages/calypso-build/babel/default.js#L17-L25